sonar-csharp-plugin/pom.xml

```diff
@@ -171,66 +170,166 @@
           </execution>
         </executions>
       </plugin>
-
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-antrun-plugin</artifactId>
-        <version>1.7</version>
-        <executions>
-          <execution>
-            <id>unzip-nuget</id>
-            <phase>validate</phase>
-            <configuration>
-              <exportAntProperties>true</exportAntProperties>
-              <tasks>
-                <copy todir="${sonarAnalyzer.workDirectory}/SonarAnalyzer.CSharp">
-                  <fileset dir="${project.build.directory}/../../sonaranalyzer-dotnet/src/SonarAnalyzer.Vsix/bin/${analyzer.build.conf}/">
-                    <include name="SonarAnalyzer.dll"/>
-                    <include name="SonarAnalyzer.CSharp.dll"/>
-                    <include name="Google.Protobuf.dll"/>
-                  </fileset>
-                </copy>
-                <zip destfile="${sonarAnalyzer.workDirectory}/static/SonarAnalyzer-${project.version}.zip"
-                     basedir="${sonarAnalyzer.workDirectory}/SonarAnalyzer.CSharp"/>
-
-                <copy todir="${sonarAnalyzer.workDirectory}/SonarAnalyzer.RuleDocGenerator.CSharp">
-                  <fileset dir="${project.build.directory}/../../sonaranalyzer-dotnet/src/SonarAnalyzer.RuleDescriptorGenerator/bin/${analyzer.build.conf}/cs">
-                    <include name="rules.xml"/>
-                    <include name="profile.xml"/>
-                    <include name="sqale.xml"/>
-                  </fileset>
-                </copy>
-                <!-- SQALE XML in sslr requires a resource to read from -->
-                <copy todir="${sonarAnalyzer.workDirectory}/org/sonar/plugins/csharp">
-                  <fileset dir="${sonarAnalyzer.workDirectory}/SonarAnalyzer.RuleDocGenerator.CSharp">
-                    <include name="*.xml"/>
-                  </fileset>
-                </copy>
-
-                <echo file="${sonarAnalyzer.workDirectory}/static/version.txt" message="${project.version}" encoding="utf-8" />
-              </tasks>
-
-            </configuration>
-            <goals>
-              <goal>run</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
     </plugins>
   </build>
 
   <profiles>
     <profile>
-      <id>debug</id>
+      <id>compile-analyzer</id>
```

Comment:
Since "compile" is a verb, `compile-analyzer` sounds like it will compile the analyzer. But it won't, it's already compiled. This is a bit misleading. How about renaming this to `compiled-analyzer` or `local-analyzer` or better idea X.